src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/LocalizedStrings.resx

```diff
@@ -190,13 +190,13 @@
     <value>Microsoft .NET Upgrade Assistant v{0}</value>
   </data>
   <data name="UpgradeAssistantHeaderDetails" xml:space="preserve">
-    <value>Makes a best-effort attempt to upgrade .NET Framework projects to current, preview or LTS versions of .NET.
+    <value>Makes a best-effort attempt to upgrade .NET Framework projects to current, preview, or LTS versions of .NET.
  
-This tool does not completely automate the upgrade process and it is expected that projects will have build errors after the tool runs. Manual changes will be required to complete the upgrade to .NET 5.
+This tool does not completely automate the upgrade process and it is expected that projects will have build errors after the tool runs. Manual changes will be required to complete the upgrade to .NET.
 
```

Comment:
maybe we should call out here to be "new versions of .NET" instead of just .NET

Reply:
I'm not sure what that adds really. What's a new version vs an old version? in 10 months before the new .NET comes out, is the previous release no longer new? 

Actually it could just be changed to:

> Manual changes will be required to finish the upgrade.

Reply:
`Manual changes will be required to finish the upgrade.` makes sense